html/apps/tweetfeed/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  background-color: gray !important;
+}
+
+md-content.md-default-theme, md-content {
+    color: rgba(0,0,0,0.87);
+    background-color: rgb(66, 66, 66);
+}
+
+img.md-user-avatar{
+  border-radius: 25%;
+}
+
+md-input-container{
+  margin-top: 3%;
+}
+
+#content-scroller{
+  overflow: visible;
+}
+
+md-toolbar button.md-button.md-icon-button:first-child {
+  margin-left: 0px;
+  margin-right: 16px;
+}
+
+md-icon{
+  height: 20px !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2597868539/issues/source?bid=3288751&fileBranchId=3307409#l28)